src/registry.js

```diff
@@ -0,0 +1,296 @@
+"use strict";
```

Comment:
But this sort of polyfill modifies the DOM environment. Normally it would modify window and window.document, but since there is no such globals, it modifies domino instead. In other words, modifying domino is a primary concern of the polyfill.